src/watertap_contrib/reflo/core/solar_energy_base.py

```diff
@@ -111,3 +215,155 @@ def calculate_scaling_factors(self):
 
         if callable(self._scaling):
             self._scaling(self)
+
+    def create_rbf_surrogate(
+        self,
+    ):
+
+        # Capture long output
+        stream = StringIO()
+        oldstdout = sys.stdout
+        sys.stdout = stream
+
+        # Create PySMO trainer object
+        self.trainer = PysmoRBFTrainer(
+            input_labels=self.input_labels,
+            output_labels=self.output_labels,
+            training_dataframe=self.data_training,
+        )
+
+        # Set PySMO options
+        # TODO: make this CONFIG for base?
+        self.trainer.config.basis_function = "gaussian"  # default = gaussian
+        self.trainer.config.solution_method = "algebraic"  # default = algebraic
+        self.trainer.config.regularization = True  # default = True
```

Comment:
For me, it makes more sense to include args for these in `create_rbf_surrogate`, but the more flexibility we enable, the more confident we should be in the model functionality/stability. So I guess it makes sense to keep as a TODO note for the time being (and maybe an issue if we don't have one yet?).

Reply:
Yes that is why I had it as a TODO for now. Get over the hurdle of this initial implementation and then refine.